File: src/test/java/kitchenpos/tobe/fixture/ProductFixture.java
package kitchenpos.tobe.fixture;

import kitchenpos.tobe.product.domain.Product;
import kitchenpos.tobe.product.domain.ProductName;
import kitchenpos.tobe.product.domain.ProductPrice;

import java.math.BigDecimal;

public class ProductFixture {

    public static Product createProduct(final String name, final BigDecimal price) {
        return new Product(new ProductName(name), new ProductPrice(price));
    }

}

File: src/test/kotlin/kitchenpos/utils/ProductFixture.kt
package kitchenpos.utils

import kitchenpos.product.tobe.domain.ProductName
import kitchenpos.product.tobe.infra.FakeProfanities

class ProductFixture {
    companion object {
        val PRODUCT_NAME = ProductName(FakeProfanities(), "양념치킨")
    }
}
